types/block.go

```diff
@@ -1082,10 +1085,16 @@ func BlobFromProto(p *tmproto.Blob) Blob {
 	if p == nil {
 		return Blob{}
 	}
+
+	if p.NamespaceVersion > math.MaxUint8 {
+		return Blob{}
+	}
```

Comment:
Do we want the overflow check for `ShareVersion` as well?

Reply:
resolved in https://github.com/celestiaorg/celestia-core/pull/985/commits/809ceb5da86339f8798a32c70f3577619f836e38